ttt.py

```diff
@@ -35,7 +36,7 @@ def transcribe_call(destinations):
         # Send the json and audiofile to a function to transcribe
         # If TTT_DEEPGRAM_KEY is set, use deepgram, else
         # if TTT_WHISPER_URL is set, use whisper.cpp else
-        # fasterwhisper deepgram_key := whispercpp_url :=
+        # fasterwhisper
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **question (code_clarification):** Clarification needed on the removal of environment variable checks.

The removal of checks for `TTT_DEEPGRAM_KEY` and `TTT_WHISPERCPP_URL` seems intentional, but it's unclear how `fasterwhisper` is now being selected without these. Could you clarify?